### PR TITLE
Update Pydev version

### PR DESCRIPTION
**Pull request by nabilbendafi**
An update to more recent version of Pydev

### CODE REVIEW

**Comment by powlo** [comment]
Are there issues with platform paths? It's a while back now but I thought I tested on both Windows and Linux without issue. I was always wary of tweaking the code pulled from pydev, because it becomes something you have to maintain.

**Comment by nabilbendafi** [comment]
I totally agree with you. I think that under Linux (which I'm using) it shouldn't be a problem. 
Anyway [Kodi Wiki](http://kodi.wiki/view/How-to:Debug_Python_Scripts_with_Eclipse#Modify_PyDev_source_in_order_to_support_XBMC_paths) says do to so.
Instead of updating the code, we can leave it to the user. Maybe adding a note in Readme should be more than enough.

**Comment by powlo** [comment]
When I was using script.module.pydevd I _was_ able to set breakpoints in my own scripts without modifying pydev source. Not sure if there's a corner case I'm not aware of, or if the wiki is out if date.

I'm not doing xbmc dev at the moment. Would you be able to test and confirm that xbmc.translatePath() really is needed?

**Comment by nabilbendafi** [comment]
Defined in my addon.xml:

```
    <import addon="script.module.pydevd" version="4.4.0"/>
```

Defined in my addon.py:

```
    import pydevd
    pydevd.settrace('chromebook', stdoutToServer=True, stderrToServer=True)
```

Installed Pydev under Kodi's addon path:

```
    <addon id="script.module.pydevd" name="pydevd" version="4.4.0" provider-name="Fabio Zadrozny, Paul Backhouse">
```

Version installed on host debugger:
- Eclipse 3.8.1
- Pydev 4.4.0.201510052309

Test on Kodi running under Linux (same machine as debugger):
- Remote connection established: OK
- Stop after settrace: OK
- Add breakpoint insertion: OK

Test on Kodi running under ATV2
- Remote connection established: OK
- Stop after settrace: OK
- Add breakpoint insertion: NOK

With or without xbmc.translatePath, when doing **remote** debugging, if PATHS_FROM_ECLIPSE_TO_PYTHON is not properly set, adding breakpoint fails
_Kodi.log on ATV_ 

```
09:54:04 T:162631680   ERROR: pydev debugger: Unable to find real location for: /home/nabil/.kodi/addons/plugin.video.animeseed/addon.py
09:54:04 T:162631680   ERROR: pydev debugger: warning: trying to add breakpoint to file that does not exist: /home/nabil/.kodi/addons/plugin.video.animeseed/addon.py (will have no effect)
```

_Eclipse Console_

```
pydev debugger: warning: trying to add breakpoint to file that does not exist: /home/nabil/.kodi/addons/plugin.video.animeseed/addon.py (will have no effect)
```

If I update my _pydevd_file_utils.py_ 

```
PATHS_FROM_ECLIPSE_TO_PYTHON = [
(r'/home/nabil/.kodi/addons/', r'/private/var/mobile/Library/Preferences/Kodi/addons/'),
(r'/home/nabil/workspace/', r'/private/var/mobile/Library/Preferences/Kodi/addons/')]
```

then it's working

```
pydev debugger: replacing to server: /home/nabil/.kodi/addons/plugin.video.animeseed/addon.py
pydev debugger: sent to server: /var/mobile/Library/Preferences/Kodi/addons/plugin.video.animeseed/addon.py
```

So in conclusion, as you said no need to touch sources with _xbmc.translatePath_, as long a Kodi is running on the same machine.

Hope it's clean enough
